src/handlers.py

```diff
@@ -150,6 +169,35 @@ async def catch_receipt(message: types.Message):
         await message.answer("Ой, как же больно! Что-то сломалось внутри меня.")
 
 
-async def get_receipts(message: types.Message):
+async def get_email_for_sending(message: types.Message):
+    await SendingMail.email.set()
+    await message.answer("Пожалуйста, напишите e-mail для отправки файла: ")
+
+
+async def get_receipts(message: types.Message, state: FSMContext):
+    pattern = r"^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$"
+    if re.match(pattern, message.text) is None:
+        await message.answer("Пожалуйста, напишите e-mail в формате user@example.com")
+        return
+    await state.update_data(email=message.text)
+    user_data = await state.get_data()
+    await state.finish()
+
     driver = DBDriver()
-    json = driver.get_receipts()
+    status = driver.add_email_for_sending(user_data["email"])
+    if status == STATUS_FAIL:
+        await message.answer("Ой, как же больно! Что-то сломалось внутри меня.")
+    else:
+        if status == STATUS_OK:
+            await message.answer("E-mail добавлен в базу")
+        elif status == STATUS_MAIL_ALREADY_EXIST:
+            await message.answer("E-mail уже есть в базе")
+        mail = driver.get_email_for_sending(user_data["email"])
+        json = driver.get_receipts()
```

Comment:
Change name of this variable. Otherwise, we could get into trouble. _json_ is python package.